### ISSUE TITLE
Set Configuration Locked but not the Tables

### ISSUE THREAD

**Issue by semighoti** [nocodb/nocodb#5374]
It would be great if the configuration can get locked but not the tables, so I can create a new Grid, filter and sort out everything, and lock it so certain people can get in and work with the custom Grids.

![Screenshot from 2023-03-27 15-01-26](https://user-images.githubusercontent.com/76666786/228076998-4205c0ad-c255-45a4-b2d2-4fb361ef1154.png)

**Comment by dstala** [comment]
@semighoti 

This mode is like an intermediate stage between locked view & collaborator view. Doesn't shared base with `editor` access serve your purpose? Shared base editors can play with tool bar options (filter, sort, etc) without hindering main project configured items.

**Comment by salim-b** [comment]
> Doesn't shared base with `editor` access serve your purpose? Shared base editors can play with tool bar options (filter, sort, etc) without hindering main project configured items.

@dstala I don't think this is what the OP meant. Currently, when one base team member edits field config like show/hide and order, it is applied globally, i.e. for all other team members. It would be great if the base owner could restict this if desired, i.e. lock field configuration, so the layout is fixed for the whole team and can only be changed by the base owner.

**Comment by dstala** [comment]
Locked views serve this purpose. Reopen if you do not agree on this & need more sophesticated handling.
